src/modules/assets/assets.service.ts

```diff
@@ -183,8 +183,8 @@ export class AssetsService {
       async (identity: string, owner: string, at: BigNumber) => {
         const atAsNumber = at.toNumber();
         const assetDto = await AssetDto.create({
-          id: getDIDFromAddress(identity),
-          owner: getDIDFromAddress(owner),
+          id: new DID(identity).did,
```

Comment:
I think it would be safer to be consistent with this by using either `getDIDFromAddress` or `new DID`, but not both. It is not strictly related to this PR though, so we can change this in other PR